### ISSUE TITLE
Missing COPYING file

### ISSUE THREAD

**Issue by AladW** [vodik/pkgelf#1]
environ/repose use GPL2, but a COPYING file is missing for pkgelf

**Comment by vodik** [comment]
Right, I need to think about how permissive I want to be with this and what I'm hoping it'll do and how much I care about maintaining/hacking on this.

Opinions/arguments?

**Comment by AladW** [comment]
It's <500 lines of code, so maybe MIT?

https://www.gnu.org/licenses/license-list.html#X11License
